packages/client-python/src/rocketride/cli/commands/store.py

```
@@ -61,393 +42,140 @@
-        return 0
-
-    async def _get_all_templates(self, client: 'RocketRideClient') -> int:
-        """
-        List all templates using the client's get_all_templates method.
-
-        Returns:
-            Exit code: 0 for success, 1 for errors
-        """
-        # Call client mixin method
-        result = await client.get_all_templates()
+                with open(file_path, 'rb') as f:
```

Ruff (0.15.7)

[warning] Async functions should not open files with blocking methods like `open`

(ASYNC230)